errorhandling/src/main/kotlin/no/nav/tms/common/errorhandling/http/StatusPages.kt

```diff
@@ -0,0 +1,37 @@
+package no.nav.tms.common.errorhandling.http
+
+import io.ktor.http.*
+import io.ktor.server.application.*
+import io.ktor.server.plugins.statuspages.*
+import io.ktor.server.request.*
+import io.ktor.server.response.*
+import no.nav.tms.common.errorhandling.logging.LoggableException
+import no.nav.tms.common.errorhandling.logging.TmsLog
+import no.nav.tms.common.errorhandling.logging.TmsSecureLog
+import no.nav.tms.common.errorhandling.redactedMessage
+
+
+fun Application.installTmsStatusPages(secureLog: TmsSecureLog, log: TmsLog) {
```

Comment:
Skulle ikke denne være extensible? Nå har man ikke mulighet til å legge til håndtering for andre typer exceptions.